### PR TITLE
[API-12254] - ClaimsApi - Update Release Notes

### PR DESCRIPTION
**Pull request by jw81**
### Description
- Updates Benefits ClaimsApi Release Notes to mention that individual POA codes for test user representatives Tamara Ellis and John Doe have been updated.

### CODE REVIEW

**Review by gileswells — DISMISSED** [review]

**Review by SlakrHakr — APPROVED** [review]

**Review by SlakrHakr — DISMISSED** [review]

File: src/content/apiDocs/benefits/claimsReleaseNotes.mdx [hunk 1]
```diff
@@ -1,3 +1,11 @@
+### February 7, 2022
+
+- Updated test user representative Tamara Ellis' individual POA code to `067`. [#320](https://github.com/department-of-veterans-affairs/vets-api-clients/pull/320)
```
[the remark this answers is not in the document]
Comment (jw81) [reply src/content/apiDocs/benefits/claimsReleaseNotes.mdx:3]: @joeabraun I'll definitely default to your wording, but I did want to point out one minor thing.  Tamara has both an _individual_ POA code and a set of _organizational_ POA codes.  In this case it is the _individual_ POA code that is being updated.

File: src/content/apiDocs/benefits/claimsReleaseNotes.mdx [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+-- Tamara Ellis' previous individual code of `A1Q` can still be used, but attempts to assign `A1Q` will not persist.  Using `067` will persist the POA change.
+- Updated test user representative John Doe's individual POA code to `072`.  [#320](https://github.com/department-of-veterans-affairs/vets-api-clients/pull/320)
```
[the remark this answers is not in the document]
Comment (jw81) [reply src/content/apiDocs/benefits/claimsReleaseNotes.mdx:5]: @joeabraun I'll definitely default to your wording, but I did want to point out one minor thing.  John Doe has both an _individual_ POA code and a set of _organizational_ POA codes.  In this case it is the _individual_ POA code that is being updated.

[the remark this answers is not in the document]
Comment (joeabraun) [reply src/content/apiDocs/benefits/claimsReleaseNotes.mdx:5]: @jw81 let's keep the individual in there then!

**Review by joeabraun — COMMENTED** [review]
Responded to @jw81 comments, and am leaving in "individual" for POA codes.

**Review by SlakrHakr — APPROVED** [review]

**Pull request merged**